OSVR-Unity/CHANGES.md

```diff
@@ -5,6 +5,18 @@ This is an abbreviated changelog for the OSVR Unity Plugin.
 Use git for a full changelog.
 ##Recent Changes
 
+### InterfaceAdapter update
+> 07-July-2015 - v0.1-199-g497cc10
```

Comment:
Not sure how you ended up with that version, maybe you missed pulling the tag, but what I had gotten earlier was `v0.2-23-gc3c5dc0`